### DISCUSSION TITLE
Integration log drain return logs for all projects even with only one project specified.

### DISCUSSION THREAD

**Discussion by jarleriksen** [vercel/community#7062] [Help] [answered]
### Summary

I have created an integration log drain using the `https://api.vercel.com/v2/integrations/log-drains` endpoint. I'm only specifying a single project id in the `projectIds` field. If I query the `https://api.vercel.com/v2/integrations/log-drains` endpoint using a GET, I correctly get the response like
```
{
       "projectIds": [
            "MY_PROJECT"
        ],
        ...
}
```
If I modify the permissions for the integration to only include one project, it'll show correctly but even if I then allow the integration to access all projects, it'll revert back to include all projects.

What am I missing here?

Thanks!

### Steps to Reproduce

Requires a team with multiple projects where the integration has access to all projects.

Create an integration log drain using a POST request to `https://api.vercel.com/v2/integrations/log-drains` endpoint with the required body.
```json
{
  "name": "Integration log drain",
  "url": "MY_URL",
  "projectIds": [
    "MY_PROJECT"
  ],
  "deliveryFormat": "json",
  "events": [
    "lambda",
    "build",
    "edge",
    "static",
    "external"
  ],
  "secret": "MY_SECRET"
}
```
The Vercel UI will show the message below in my case.
<img width="317" alt="Screenshot 2024-06-02 at 19 09 58" src="https://github.com/vercel/community/assets/25452489/df13c372-e676-4a8c-a1dc-96ed526d47b7">

If you do a GET request to `https://api.vercel.com/v2/integrations/log-drains`, it'll show that the log drain is only associated to one project.

**Comment by jarleriksen** [comment]
Still having issues with this

**Reply by ghost** [reply]
i trying to fix it

**Comment by amyegan** [comment, marked as answer]
Thanks for also posting this report in https://vercel.community/t/integration-log-drain-return-logs-for-all-projects-even-with-only-one-project-specified/318

The team released a fix so this should no longer be a problem 🙂
